doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -267,19 +273,20 @@ monitors::
       2: 127.0.0.1:6795/0 mon.c
       
 This may not be what you have however. For instance, in some versions of
-early Cuttlefish there was this one bug that could cause your ``monmap``
+early Cuttlefish there was a bug that could cause your ``monmap``
 to be nullified.  Completely filled with zeros. This means that not even
-``monmaptool`` would be able to read it because it would find it hard to
-make sense of only-zeros. Some other times, you may end up with a monitor
-with a severely outdated monmap, thus being unable to find the remaining
+``monmaptool`` would be able to make sense of cold, hard, inscrutable zeros.
```

Comment:
fancy, fancy. Let's leave this in. It has dash.

Reply:
Hah.  I am at times influenced by this classic: http://www.catb.org/~esr/jargon/html/story-of-mel.html

Reply:
I haven't thought of the story of Mel in a couple of years.